tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+## Content of this article
+
+[Step 1: Install MeiliSearch](#Step-1-Install-MeiliSearch)
+
+[Step 2: Run MeiliSearch as a service](#Step-2-Run-MeiliSearch-as-a-service)
++ [2.1 Create a service file](#21-Create-a-service-file)
++ [2.2. Enable and start service](#22-Enable-and-start-service)
+
+[Step 3: Secure and finish your setup. Using a Reverse Proxy, domain name and HTTPS](#Step-3-Secure-and-finish-your-setup-Using-a-Reverse-Proxy-domain-name-and-HTTPS)
++ [3.1. Creating a Reverse Proxy with Nginx](#31-Creating-a-Reverse-Proxy-with-Nginx)
++ [3.2. Activating SSL (HTTPS) on your MeiliSearch](#32-Activating-SSL-HTTPS-on-your-MeiliSearch)
+
+[Conclusion](#Conclusion)
+  
+## Get your MeiliSearch ready for production
+  
+For this tutorial, we will be using a Debian 10 server, running on DigitalOcean. You can easily try it on your own, with plans starting at $5/month. And if you want some credits to start running your MeiliSearch and are not already registered on DigitalOcean, you can get $100 for free using [this referral link](https://m.do.co/c/7c67bd97e101).  
+
+
+
+## Prerequisites
+
+- An up-to-date server that runs Debian 10
+- An ssh keypair to connect to that machine
+
+> *TIPS:* learn how to connect via SSH to your [DigitalOcean droplet](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/) or any [Linux or windows server](https://phoenixnap.com/kb/ssh-to-connect-to-remote-server-linux-or-windows)
+
+## Step 1: Install MeiliSearch
+
+Installing and running MeiliSearch is an easy and straightforward process. In order to keep this tutorial as simple as possible, we are going to use a MeiliSearch installation script that will take care of the process for you. It will get a copy of a MeiliSearch binary in your machine and let you use it inmediatley.
+
+Once we are logged in into our machine via SSH, we only need to be sure that our system and its dependencies are up-to-date and install MeiliSearch.
+```bash
+# Update references to the dependencies on Debian
+$ apt update
+# Install curl, that we need to install MeiliSearch in the next step
+$ apt install curl -y
+# Install MeiliSearch latest version from the script
+$ curl -L https://install.meilisearch.com | sh
+```
+
+> The diferent options to achieve a MeiliSearch install are detailed in our **[Installation Docs](https://docs.meilisearch.com/guides/advanced_guides/installation.html#download-and-launch)**
+
+> It is important to know that there are different ways to get MeiliSearch running on your machine. As an open source project, you can always compile the latest stable release of MeiliSearch from the source code to ensure the binary uses your achitecture in the best possible way.  
```

Comment:
Maybe I'm wrong. Not sure about the meaning.
```suggestion
> It is important to know that there are different ways to get MeiliSearch up and running. It is possible to compile the latest stable release of MeiliSearch from the source code to ensure the binary remains adapted to your achitecture.
```

Reply:
```suggestion
> It is important to know that there are different ways to get MeiliSearch running on your machine. As an open source project, you can always compile the latest stable release of MeiliSearch from the source code to ensure the binary uses your achitecture in the best possible way.  
```

Reply:
The idea is that compiling it in your own environment let you have some compiler optimization that might be possible for your specific architecture. Getting the 'already compiled' binary wouldn't have those optimizations, but is easy and straightforward, which is why we are doing it that way in this tutorial.

Reply:
Noted! I cancel this one.